src/engine/source/metrics/include/metrics/dataHubExporter.hpp

```diff
@@ -0,0 +1,87 @@
+#ifndef __JSON_EXPORTER_H
+#define __JSON_EXPORTER_H
```

Comment:
Adjust to the style, check the other files:
```suggestion
#ifndef _METRICS_DATAHUB_EXPORTER_H
#define _METRICS_DATAHUB_EXPORTER_H
```